test/test-jsdoc.js

```diff
@@ -1,6 +1,93 @@
 var test = require("tape");
 require("../index");
 
+test("Check inline markdown", assert => {
+	let text = `
+* *strong*
+* __emphasis__
+* @light@              -> <code>light</code> 
+* @light@  @light@            -> <code>light</code> <code>light</code>
+* {@link Resource}     -> link?
+* @ignore(qx.*)
+`  
+  var test = qx.tool.compiler.jsdoc.Parser.parseComment(text);
+  console.log(test["@description"][0].body);
```

Comment:
This doesn't really test anything, does it? Maybe you could save the output in a file and use the content as the "expected" value?

Reply:
Next time